packages/astro/src/core/create-vite.ts

```diff
@@ -196,7 +196,7 @@ export async function createVite(
 		const applyToFilter = command === 'build' ? 'serve' : 'build';
 		const applyArgs = [
 			{ ...settings.config.vite, mode },
-			{ command, mode },
+			{ command: command === 'dev' ? 'serve' : command, mode },
```

Comment:
I was following the pattern at here too:

https://github.com/withastro/astro/blob/1f749c942af4fcf138f9c4be07ce6b189c0a88fa/packages/astro/src/config/index.ts#L12-L13

I think it's unlikely we'll change the command name so it might be fine 🤔